compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/SkiaTextInputService.skiko.kt

```diff
@@ -23,7 +23,10 @@ import androidx.compose.ui.text.input.ImeOptions
 import androidx.compose.ui.text.input.PlatformTextInputService
 import androidx.compose.ui.text.input.TextFieldValue
 
-internal class SkiaTextInputService : PlatformTextInputService {
+internal class SkiaTextInputService(
+    private val _showSoftwareKeyboard: () -> Unit,
```

Comment:
To avoid "_" in "public" API, it is better to write this way:
```
internal class SkiaTextInputService(
    showSoftwareKeyboard: () -> Unit,
) {
   private val _showSoftwareKeyboard = showSoftwareKeyboard
```